features/orders.feature

```diff
@@ -1261,3 +1261,27 @@ Feature: Orders
         "channel":"coopcycle_order_events#1"
       }
       """
+
+  @debug
```

Comment:
@alexsegura should we remove this tag?
BTW. The step `And the user "bob" has ordered something at the restaurant with id "1"` is breaking in my local for all the orders tests :thinking:

Reply:
> The step And the user "bob" has ordered something at the restaurant with id "1" is breaking in my local

We need to investigate this, no idea why.